cicecore/cicedynB/infrastructure/ice_read_write.F90

```diff
@@ -1106,14 +1106,15 @@ subroutine ice_read_nc_xy(fid,  nrec,  varname, work,  diag, &
       integer (kind=int_kind) :: & 
          varid          , & ! variable id
          status             ! status output from netcdf routines
```

Comment:
When uncommenting the 'optional diagnostics' in this routine, multiple declaration lines in this subroutine require adding an & at the end of the line. 

I believe a better solution would be to add the additional variables as separate lines which could be cleanly uncommented when diagnostic output is desired, eg:

!  integer (kind=int_kind) :: &
!  ndim, vnvar, & ! sizes of netcdf file
! id,                   & ! dimension index
! dimlin             & ! dimension size